notanorm/mysql.py

```diff
@@ -179,10 +180,19 @@ def create_table(self, name, schema):
                 index_name = "ix_" + name + "_" + "_".join(idx.fields)
                 unique = "unique " if idx.unique else ""
                 icreate = "create " + unique + "index " + index_name + " on " + name + " ("
-                icreate += ",".join(idx.fields)
+                for f, f_len in zip(idx.fields, idx.field_lengths):
+                    print(f if f_len is None else f"{f}({f_len})")
+                if idx.field_lengths:
```

Comment:
should always have this, right? just None's?

Reply:
No longer relevant with the `DbIndexField` change